src/__tests__/__utils.ts

```diff
@@ -0,0 +1,126 @@
+import fetch from 'node-fetch';
+import { execute, toPromise } from 'apollo-link';
+import { ApolloServer } from 'apollo-server-lambda';
+import { GeoDataSource } from '../graphql/dataSources/geo/GeoDataSource';
+import { StorageDataSource } from '../graphql/dataSources/storage/StorageDataSource';
+import resolvers from '../graphql/resolvers';
+import typeDefs from '../graphql/schemas/schemas';
+import { spawn } from 'child_process';
+import cwd from 'cwd';
+import { GraphQLClient } from "graphql-request";
+
+export const toPromiseExecution = toPromise;
+
+export const mockedContext = {
+    geoClient: {
+      forwardGeocode: jest.fn()
+    },
+    db: {
+      scan: jest.fn(),
+      put:  jest.fn(),
+      update: jest.fn(),
+      delete: jest.fn()
+    }
+};
+
+/**
+ * Integration testing utils
+ */
+export const constructTestServer = () => {
+  const geoDataStorage = new GeoDataSource(mockedContext.geoClient);
+  const storeDataStorage = new StorageDataSource(mockedContext.db);
+
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+    context: ({ event, context }) => ({
```

Comment:
How do you decide whether to use a callback or async/await?

Reply:
I prefer to use async/await as much as possible. It makes the code cleaner and more straight forward.